sphinx/users/imagej/index.rst

```diff
@@ -27,7 +27,7 @@ ways:
    `Image5D <http://imagej.net/Image5D>`_ plugin
    (if installed)
 -  With Rainer Heintzmann's
-   `View5D <http://www.nanoimaging.de/View5D/>`_ plugin (if installed)
+   `View5D <https://github.com/fiji/View5D/>`_ plugin (if installed)
```

Comment:
Another possible option might be https://imagej.net/plugins/view5d which has the benefit to linking to both the source code repository and the (currently broken) homepage